### PR TITLE
[RAPPS-DB] Add Minecraft 4K

### PR DESCRIPTION
**Pull request by fiatcoupe16vturbo**
image

i mirrored from: https://maverk-corporation.itch.io/minecraft4k

### CODE REVIEW

File: minecraft4k.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Minecraft 4K
+Version = v2
+License = Apache
+Description = Minecraft 4k was an edition of Minecraft developed for the 2010 Java 4K contest.
+Category = 4
+URLSite = https://maverk-corporation.itch.io/minecraft4k
+URLDownload = https://github.com/fiatcoupe16vturbo/Minecraft-4K-Mirror/raw/refs/heads/main/Minecraft4K.jar
+SHA1 = 49f7ae310b48dd41266a2e01c91cfb5325dda0ee
+SizeBytes = 16384
```
Comment (JoachimHenze) [inline minecraft4k.txt:10]: I do doubt that Minecraft would be only 16kByte in size.

Comment (JoachimHenze) [reply minecraft4k.txt:10]: If I download that file, then the size is 9.852 Bytes for me:
image
So please fix that value.

File: minecraft4k.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Minecraft 4K
+Version = v2
+License = Apache
+Description = Minecraft 4k was an edition of Minecraft developed for the 2010 Java 4K contest.
```
[the remark this answers is not in the document]
Comment (fiatcoupe16vturbo) [reply minecraft4k.txt:5]: i tested this on azul zulu openjdk (java 8) so can i write this version?

> 4K

[the remark this answers is not in the document]
Comment (JoachimHenze) [reply minecraft4k.txt]: Mention a version that we do have in rapps already, and that would work.

**Review by JoachimHenze — APPROVED** [review]
Approved

**Pull request merged**